docs/enterprise-guide/whitelabeling.md

```diff
@@ -6,29 +6,84 @@ title: White labeling Metabase
+- [Ubuntu](https://fonts.google.com/specimen/Ubuntu)
 
-You can replace Metabase’s familiar, tasteful, inspired-yet-not-threateningly-avant-garde dotted M logo with your very own. For things to work best, the logo you upload should be an SVG file that looks good when it’s around 60px tall. (In other words, ask the nearest designer for help.)
+You might need to refresh your browser to see your changes take effect.
 
-## Change the color palette
+## Color palette
+
+You can customize colors in both the application UI and in the Metabase charts.
+
+### User interface colors
+
+![User interface colors](images/appearance/user-interface-colors.png)
 
 You can customize the colors that Metabase uses throughout the app:
 
-- **Primary color:** by default, this is the lovely blue that is used for the nav bar, links, buttons, and more.
-- **Nav bar color:** you can optionally specify a separate color for the nav bar; otherwise it defaults to the Primary color.
-- **Accent colors:** these are the colors that are used for some icons, secondary buttons, elements of the graphical query builder, and chart colors.
-- **Additional chart colors:** as the name implies, these are additional colors that are included in the charting options to give your lines, bars, and pie slices some extra pizazz.
+- **First color:** The main color used throughout the app for buttons, links, and the default chart color.
+- **Second color:** The color of aggregations and breakouts in the graphical query builder.
+- **Third color:** Color of filters in the query builder, buttons and links in filter widgets.
 
-To learn more, check out this article on [branding your Metabase instance](https://www.metabase.com/blog/white-label/index.html).
+### Chart colors
 
----
+![Chart colors](images/appearance/chart-colors.png)
 
-## Next: Copying contents of one Metabase instance to another
+You can choose up to 24 hex values. If you choose fewer than 24 colors, Metabase will auto-generate colors to fill in the rest of the values.
 
-Learn how to use [serialization](serialization.md) to create and load data dumps of the contents of a Metabase instance.
+## Logo
+
+You can replace Metabase’s familiar, tasteful, inspired-yet-not-threateningly-avant-garde dotted M logo with your very own logo. For things to work best, the logo you upload should be an SVG file that looks good when it’s around 60px tall. (In other words, ask the nearest designer for help.)
+
+## Favicon
+
+The url or image that you want to use as the favicon. Note that if you use a relative path, that path isn't relative to the Metabase JAR, but to the webserver. So unless you're using a reverse-proxy, the path will be relative to the frontend resources available to the JAR.
+
+## Landing page
+
+The landing page is what people will see whenever they login. You can set the URL to a collection, question, dashboard or whatever, just make sure that everyone has access to that URL.
```

Comment:
A natural follow-up question from the reader: can I set different landing pages based on user attributes or groups? It might be a good idea to include a note about that here.